workflow-templates/im-build-npm-package.yml

```diff
@@ -196,14 +196,27 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
           script: |
-            let nextVersion = '${{ needs.build-and-publish-to-gpr.outputs.NEXT_VERSION }}';
-            let hasRelease = nextVersion && nextVersion.length > 0;
-            let tagText = hasRelease ? 
-              '[Tag - ${{ needs.build-and-publish-to-gpr.outputs.NEXT_VERSION }}](${{ env.REPO_URL }}/releases/tag/${{ needs.build-and-publish-to-gpr.outputs.NEXT_VERSION }})' : 
-              'Tag - N/A';
+            let nextVersion = '${{ env.NEXT_VERSION }}';
+            let isMergeToMain = '${{ env.IS_MERGE_TO_MAIN }}' == 'true';
```

Comment:
I remember these two env variables were set up for the previous job, but are they for this one?

This is actually something I was going to comment that we should add to these templates while we were doing today's migration, but I didn't want to interrupt. I'm glad you thought of it too!

Reply:
I'm making the same change in too many places.  I saw your comment and was like, I added that.  I added it to my start-project PR not this PR.  grrr!

Reply:
I'm glad you caught that.  Those changes have been pushed up (for real this time).